### PR TITLE
fix styles in my orcid

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

File: src/app/record/components/work-stack/work-stack.component.scss [hunk 1]
```diff
@@ -17,3 +17,7 @@
     position: relative;
   }
 }
+
+a {
```
Comment (leomendoza123) [inline src/app/record/components/work-stack/work-stack.component.scss:21]: Should we maybe create a global style with this?  here https://github.com/ORCID/orcid-angular/blob/dev/src/styles.scss#L59-L69

File: src/app/record/components/funding/funding.component.html [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
 <!-- GENERAL DATA DISPLAY BY DEFAULT -->
 <app-panel-data
-  class="defaul-data-loaded orc-font-body-small"
+  class="default-data-loaded orc-font-body-small"
   [borderBottom]="true"
 >
   <div>
     <div class="general-data">
-      <ng-container *ngIf="funding.startDate.year"
+      <ng-container *ngIf="funding && funding.startDate"
```
Comment (leomendoza123) [inline src/app/record/components/funding/funding.component.html:10]: We might use conditional chaining for a cleaner look. 
But this is fine, no need to change

**Review by leomendoza123 — CHANGES_REQUESTED** [review]
Just add the URL underline as a global class

File: src/app/record/components/work-stack/work-stack.component.scss (hunk 1 above)
Comment (DanielPalafox) [reply src/app/record/components/work-stack/work-stack.component.scss]: Done!

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**